src/explorer.ts

```diff
@@ -70,12 +73,13 @@ export class KubernetesExplorer implements vscode.TreeDataProvider<KubernetesObj
 
     private async getClusters(): Promise<KubernetesObject[]> {
         const contexts = await kubectlUtils.getContexts(this.kubectl);
-        return contexts.map((context): KubernetesContext => {
+        return contexts.map((context): ClusterContext => {
```

Comment:
This type declaration seems wrong (and seems like it was wrong before).  `kubectlUtils.getContexts` returns `kubectlUtils.KubectlContext[]` - a model object not an explorer object.  (But funnily enough it seems like TypeScript is ignoring the type declaration and inferring `context` as `KubectlContext`... weird...)

Reply:
It seems like it's inferring the correct type for me…